RELEASE/scripts/autoscend/quests/level_08.ash

```diff
@@ -277,6 +277,10 @@ boolean L8_getGoatCheese()
 	{
 		auto_log_info("Attemping to use Map the Monsters to olfact a Dairy Goat.");
 	}
+	if (my_fullness() > 0 && auto_haveAugustScepter() && get_property("_augSkillsCast").to_int() < 5 && !get_property("_aug16Cast").to_boolean())
```

Comment:
this same block is in many locations. I'd suggest making a new function for. I in mr2023 and call that instead. Better to only have one spot to change logic, if a change is needed

Perhaps could update `auto_lostStomach()` to `auto_lostStomach(boolean forceUsage)` If forceUsage is false, then wait for day end. If true, don't wait for day end. Then have one function for both use cases

Reply:
That's a good idea